src/cli.js

```diff
@@ -145,18 +149,68 @@ function promptDockerHubCredentials() {
 
 
 /**
+ * Make sure the user has all requirements satisfied before allowing
+ * them to continue creating a project.
+ */
+function promptGlobalPrerequisites() {
+  const questions = [];
+
+  // If this is the first time settings prereqs in the preferences, create the object
+  if (!preferences.prereqs) {
+    preferences.prereqs = {};
+  }
+
+  constants.hammer.globalPrereqs.forEach((prereq) => {
+    // Only ask them if they haven't said YES in the past (i.e. if it's not complete)
+    if (!preferences.prereqs[prereq.name]) {
+      questions.push({
+        name: prereq.name,
+        type: 'confirm',
+        message: prereq.message,
+        default: false
+      });
+    }
+  });
+
+  return inquirer.prompt(questions);
+}
+
+export function userIsFinishedWithPrereqs(answers) {
```

Comment:
I think I better name for this would be `isUserFinishedWithPrereqs` easier to tell this should return a boolean.

Reply:
Agreed.